test/integration/consumer_group_test.exs

```diff
@@ -7,10 +7,11 @@ defmodule KafkaEx.ConsumerGroup.Test do
 
   test "consumer_group_metadata works" do
     random_string = generate_random_string
-    pid = Process.whereis(KafkaEx.Server)
     KafkaEx.produce(%Proto.Produce.Request{topic: "food", required_acks: 1, messages: [%Proto.Produce.Message{value: "hey"}]})
     KafkaEx.fetch("food", 0, offset: 0)
-    metadata = KafkaEx.consumer_group_metadata(KafkaEx.Server, random_string)
+    KafkaEx.create_worker(:consumer_group_metadata_worker, consumer_group: random_string, uris: Application.get_env(:kafka_ex, :brokers))
+    pid = Process.whereis(:consumer_group_metadata_worker)
+    metadata = KafkaEx.consumer_group_metadata(:consumer_group_metadata_worker, random_string)
```

Comment:
I realize this isn't actually something that you changed in this  PR, but there's at least three different "metadata"s involved in a Kafka client - consumer metadata, consumer group metadata, and offset metadata.  Variables named just "metadata" therefore carry some risk of confusion...